libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -197,22 +195,21 @@ int16_t AP_Motors6DOF::calc_thrust_to_pwm(float thrust_in) const
 
 void AP_Motors6DOF::output_to_motors()
 {
-    int8_t i;
     int16_t motor_out[AP_MOTORS_MAX_NUM_MOTORS];    // final pwm values sent to the motor
 
     switch (_spool_mode) {
     case SHUT_DOWN:
         // sends minimum values out to the motors
         // set motor output based on thrust requests
-        for (i=0; i<AP_MOTORS_MAX_NUM_MOTORS; i++) {
+        for (int8_t i = 0; i < AP_MOTORS_MAX_NUM_MOTORS; i++) {
```

Comment:
I think the variable "i" in this method is using multiple loops, so I think that it is OK as it is.